rasa/utils/endpoints.py

```diff
@@ -46,7 +46,13 @@ def concat_url(base: Text, subpath: Optional[Text]) -> Text:
     `base`."""
 
     if not subpath:
-        return base.rstrip("/")
+        if base.endswith("/"):
+            logger.debug(
+                "The URL '{}' has a trailing slash. Please make sure the "
+                "target server supports trailing slashes for this "
+                "endpoint.".format(base)
+            )
+        return base
 
     url = base
     if not base.endswith("/"):
```

Comment:
I'm not exactly sure what's going on here, should the debug message also go in this if statement?

Reply:
I decided against, cause if `subpath` values are added by Rasa code (and not externally) and we should know what we are doing and if we add a slash or not.

Reply:
Ah okay I got it, I was confused bc i was missing that these were 2 cases with and w/o subpath